afs/media/js/views/components/reports/physical-thing.js

```diff
@@ -371,6 +386,137 @@ define([
+                const digitalResourceServiceIdentifierContentNodeId = '56f8e9bd-ca7c-11e9-b578-a4d18cec433a';
+                const digitalServiceTile = digitalResourceData.tiles.find(function(tile) {
+                    return tile.nodegroup_id === digitalResourceServiceIdentifierNodegroupId;
+                });
+                return window.fetch(digitalServiceTile.data[digitalResourceServiceIdentifierContentNodeId])
+                    .then(function(response){
+                        if(response.ok) {
+                            return response.json();
+                        }
+                    });
+            };
+
+            const resourceId = ko.unwrap(self.reportMetadata).resourceinstanceid;
+            const loadRelatedResources = async() => {
+                const digitalResourceGraphId = '707cbd78-ca7a-11e9-990b-a4d18cec433a';
+                const IIIFManifestConceptId = '0c682c76-a6a4-48f0-9c5b-1203a6dc33da';
+
+                const result = await reportUtils.getRelatedResources(resourceId);
+                const relatedResources = result?.related_resources;
+                
+                const relatedDigitalResources = relatedResources.filter(resource => resource.graph_id === digitalResourceGraphId);
+                if (relatedDigitalResources.length) {
+                    relatedDigitalResources.forEach(async resource => {
+                        const resourceDomainConceptIds = resource.domains.map(x => x.conceptid)
+                    
+                        // If there is an IIIF manifest in the related resources, load the thumbnail from that manifest
+                        if (resourceDomainConceptIds.includes(IIIFManifestConceptId)) {
+                            const imageResource = await self.getThumbnail(resource);
+                            if (imageResource) {
+                                self.imageSummary(Array({
+                                    displayname: imageResource.label,
+                                    thumbnail: imageResource.sequences[0].canvases[0].thumbnail['@id']
+                                }));
+                            }
+                        }
+                    });
+                }
+            };
+
+            loadRelatedResources();
+
+            const statmentData = self.resource()?.statement;
+            if (statmentData) {
+                self.statementsSummary(statmentData.map(x => {
+                    const type = self.getNodeValue(x, 'statement_type');
+                    const content = self.getNodeValue(x, 'statement_content');
+                    const language = self.getNodeValue(x, 'statement_language');
+                    const tileid = self.getTileId(x);
+                    return { type, content, language, tileid }
+                }));
+            };
+
+            const typeData = self.resource()?.type;
+            if (typeData) {
+                self.typeSummary(Array({
```

Comment:
Two things here - you can use cheap array initialization (instead of using the array constructor - just put in brackets []).  Second, looks like you found an interesting edge case for getRawNodeValue in report.js - I'm guessing you called @display_value here because I'm guessing not doing so returned an error for you.  I think that's actually because of lines 31-33 in report.js - that they're in the array loop.  I suspect that's incorrect and it would be better to actually move those lines and line 25 outside of the loop.  Without testing it, that should fix this behavior and you won't need to specify @display_value after that.